### ISSUE TITLE
Respect excluded files Obsidian setting

### ISSUE THREAD

**Issue by tcoates3** [ulisesantana/obsidian-pending-notes#15]
Under the Files and Links section of settings, Obsidian provides the Excluded Files option, which allows you to select files or folders that Obsidian ignores for search results, graph view, etc. The plugin currently doesn't respect this, so links in excluded files or folders still show up in the pending notes list. 

Thanks for the plugin, indispensable for my vault!

**Comment by mariomui** [comment]
Yeah, i need this too. My vault is getting bigger and I'm getting a lot of false positives. So ... are you still maintaining this or can i just yoink it?

**Comment by ulisesantana** [comment]
Hi! Sorry for the delay, but I didn't see the email notification for this. The support for excluded folders is on version 0.9.0

I hope this works fine for you. I said this because Obsidian has no developer documentation, so I need to figure out how this excluded folders was implemented and try it out. At least I can say that in macOS is working.

If you find any issue don't hesitate to share it here.
